README.rst

```diff
@@ -70,9 +70,99 @@ We appreciate any help!
 BUILD & DEVELOPMENT
 -------------------
 
-Run ``python3 setup.py install``
+1. Installation instructions for **Windows** users:
 
-Note: You need to have bower installed before you may install the python packages.
+   - Clone the **coala-html** repository
+
+     ::
+
+       $ git clone https://github.com/coala/coala-html.git
+
+   - Download and install the latest version of Node.js from `official website
+     <https://nodejs.org/en/download/>`__.
+
+   - Change the directory to **coala-html** and run following command to
+     install dependencies
+
+     ::
+
+       $ python setup.py install
+
+   - Install **coala-bears**
+
+     ::
+
+       $ pip install coala-bears
```

Comment:
```suggestion
       $ pip3 install coala-bears
```

Reply:
That instruction is for windows users. 
In windows, default `pip` is being used no-matters which version of python. Even with python 3.x we can use `pip`

Reply:
With both Python 2 and 3 installed, it's possible that pip might point to `path/to/python2/Scripts/pip/executable` in some cases.

Reply:
yes